bin/getTitle.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:http/http.dart' as http;
+import 'package:html/parser.dart' show parse;
+// import 'package:html/dom.dart' hide Text;
+
+dynamic main() {
+  printTitle();
```

Comment:
Similarly to the previous exercise, could you make it so that main looks like this:

```dart
void main() {
  print(getTitle("https://example.com"));
}
```

This will make it easier later to reuse the function on another site.